frontend-customers/src/app/pages/success-page/success-page.component.html

```diff
@@ -0,0 +1,26 @@
+<h1>Buchung erfolgreich</h1>
+<p>
+    Ihre Buchung war erfolgreich. Sie können nun bis zu 30 Minuten vor dem Abflug den Check-In online durchführen. Für weitere Details besuchen Sie bitte die Übersichtseite für Ihre gebuchten Flüge.
+</p>
+
+<mat-card>
+    <mat-card-header>
+        <p>Ihr Flug von <b>{{start}}</b> nach <b>{{destination}}</b></p>
+    </mat-card-header>
+    <mat-card-content>
+        <p>Start: {{departureDate1 | date:'dd.MM.yyyy'}}, {{departureDate1 | date:'HH:mm:ss'}} Uhr</p>
+        <p>Landung: {{arrivalDate1 | date:'dd.MM.yyyy'}}, {{arrivalDate1 | date:'HH:mm:ss'}} Uhr</p>
+        <p>Flugdauer: {{getDuration(1)}}</p>
+    </mat-card-content>
+</mat-card>
```

Comment:
Durch das Array lässt sich hier die Direktive *ngFor verwenden um die HTML Tags nicht 2 mal zu schreiben

`<mat-card *ngFor="let flight of flights"> </mat-card>`